pkgs/applications/networking/cluster/ksonnet/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoPackage, fetchFromGitHub, ... }:
+
+let version = "0.11.0"; in
```

Comment:
We generally put the version attribute inside of the derivation and use a recursive attribute set (`rec` keyword).